0.10/Dockerfile

```diff
@@ -41,4 +38,4 @@ USER nodejs
 
 EXPOSE 3000
 
-CMD ["/opt/nodejs/bin/usage"]
\ No newline at end of file
+CMD ["/usr/local/bin/usage"]
```

Comment:
This should be /usr/local/sti/ ?

Also if you add this into PATH, you don't have to write absolute path.